apache/tests/compose/apache.yaml

```diff
@@ -5,6 +5,8 @@ services:
     image: "httpd:${APACHE_VERSION}"
     volumes:
       - ${APACHE_CONFIG}:/usr/local/apache2/conf/httpd.conf
+      - ${DD_LOG_1}:/usr/local/apache2/logs/access_log
+      - ${DD_LOG_2}:/usr/local/apache2/logs/error_log
```

Comment:
Related to #5346, I'm thinking we may want to define a mapping of indexes to human-readable UIDs in the spec, and then generate the environment variables from the UIDs? (These shouldn't be defined in the `example` field, but possibly at the same level as a field specific to `template: logs`.)

Eg in the spec…

```yaml
- template: logs
  # v 
  ids:  # Or 'names', or 'entries'…
    - access
    - error
  # v This remains unchanged.
  example:
  - type: file
    path: /var/log/apache2/access.log
    source: apache
    sourcecategory: http_web_access
    service: apache
  - type: file
    path: /var/log/apache2/error.log
    source: apache
    sourcecategory: http_web_access
    service: apache
```

And then here…

```yaml
volumes:
  - ${DD_LOG_ACCESS}:/usr/local/apache2/logs/access_log
  - ${DD_LOG_ERROR}:/usr/local/apache2/logs/error_log
```

Reply:
We shouldn't add new fields to the spec imo that aren't directly useful to users i.e. this would only be an implementation detail used by our tests

Reply:
Hmmm, however I would certainly be open to a new `short_name` key. Can you ask the logs team if unexpected fields would break anything?

Reply:
Rebased